packages/ui-components/src/components/Chart/Tooltip.js

```diff
@@ -132,6 +132,23 @@ const SingleValueTooltip = ({ valueType, payload, periodGranularity, labelType }
   );
 };
 
+const NoDataTooltip = ({ payload, periodGranularity }) => {
+  const data = payload[0]?.payload;
+  const { name = undefined, timestamp = undefined } = data || {};
```

Comment:
This is also where we wind up as data is loading, so need to handle the possibility of literally everything being undefined

Reply:
I think another option would be to return null if data is undefined but overall it's much of a muchness.